MySqlImport/MySqlImport.ecl

```diff
@@ -0,0 +1,151 @@
+IMPORT MySql,Std;
+
+EXPORT MySqlImport := MODULE, FORWARD
+
+    /***************************************************************************
+     * To use this bundle, execute the bundle, specificing the database, user
+     * and password on the command line:
+     *
+     * ecl run MySqlImport.ecl -xdatabase=mydb -xuser=ghalliday -xpassword=password
+     *
+     * The bundle outputs a single string as a result which contains a module
+     * that can be used to access all the tables in the MySql database 
+     *    
+     **************************************************************************/
+
+    EXPORT Bundle := MODULE(Std.BundleBase)
+        EXPORT Name := 'MySqlImport';
+        EXPORT Description := 'Create record and dataset definitions from MySQL tables';
+        EXPORT Authors := ['Gavin Halliday'];
+        EXPORT License := 'http://www.apache.org/licenses/LICENSE-2.0';
+        EXPORT Copyright := 'Copyright (C) 2015 HPCC Systems';
+        EXPORT DependsOn := [];
+        EXPORT Version := '1.0.0';
+    END;
+
+    string database := '' : stored('database');
+    string user := '' : stored('user');
+    string password := '' : stored('password');
+    
+    columnRecord := RECORD
+        string name;
+        string kind;
+        DATA fullkind;
+        unsigned maxlen;
+    END;
+    
+    tableRecord := RECORD
+        string table_name;
+        string comment;
+    END;
+    
+    //dataset(tableRecord) myTables := EMBED(mySql : database('orderella'),user('orderella'),database('orderella'))//,server('127.0.0.1'), port('3306')
+    dataset(tableRecord) myTables() := EMBED(mySql : database(database),user(user),password(password))
+        SELECT table_name, table_comment FROM information_schema.tables WHERE table_type='BASE TABLE' AND table_schema='orderella';
+    ENDEMBED;
+    
+    dataset(columnRecord) getColumns(string table) := EMBED(mySql : database('orderella'),user('orderella'),database('orderella'))
```

Comment:
This should be using the stored variables (or better, module parameters) rather than constants.